### ISSUE TITLE
Suggestion: make iOS/OSX overscroll look pretty

### ISSUE THREAD

**Issue by callumlocke** [Financial-Times/next-sass-setup#109]
The site doesn't exploit the way iOS and OSX uses the `body` element's background colour to paint the overscroll area. Especially on iPhone, it looks weird when you overscroll – the fixed menu bar at the top gets detached from the content by a wide stripe of `#ffffff`.

As both the header and footer have the same dark grey background color, it makes sense to paint the overscroll area in that same dark grey. 

I tried the making this change locally and it looks a lot nicer imo:

- move the `background: #fdf8f2;` from the `html` element to  `body > .n-layout`
- set `background: #333333` on the `body` element

**Comment by wheresrhys** [comment]
Good idea. This should be responsive though as on larger screens (M upwards I think) there's an advert on a pink background above the header, soon landscape tablets pink would make more sense
